magicctapipe/io/io.py

```diff
@@ -66,6 +58,55 @@
 DEAD_TIME_LST = 7.6 * u.us
 DEAD_TIME_MAGIC = 26 * u.us
 
+def telescope_combinations(config):
+    """
+    Generates all possible telescope combinations without repetition. E.g.: "LST1_M1", "LST2_LST4_M2", "LST1_LST2_LST3_M1" and so on.
+
+    Parameters
+    ----------
+    config: dict
+        yaml file with information about the telescope IDs. Typically evoked from "config_general.yaml" in the main scripts.
```

Comment:
Not sure about having "typical" file names in docstring

Reply:
Removed